### ISSUE TITLE
"ERROR: QuadGK: fetch failed to get commit 064d2f01a0" during Pkg.update()

### ISSUE THREAD

**Issue by goedman** [JuliaMath/QuadGK.jl#11]
Since a few days I notice below issue (both Julia 0.6 and Julia 0.7-):

ERROR: QuadGK: fetch failed to get commit 064d2f01a0, please file an issue at https://github.com/JuliaMath/QuadGK.jl/issues
Stacktrace:
 [1] fetch(::Base.LibGit2.GitRepo, ::String, ::String) at ./pkg/write.jl:25
 [2] #3 at ./pkg/write.jl:53 [inlined]
 [3] with(::Base.Pkg.Write.##3#4{String,String}, ::Base.LibGit2.GitRepo) at ./libgit2/types.jl:608
 [4] resolve(::Dict{String,Base.Pkg.Types.VersionSet}, ::Dict{String,Dict{VersionNumber,Base.Pkg.Types.Available}}, ::Dict{String,Tuple{VersionNumber,Bool}}, ::Dict{String,Base.Pkg.Types.Fixed}, ::Dict{String,VersionNumber}, ::Set{String}) at ./pkg/entry.jl:542
 [5] update(::String, ::Set{String}) at ./pkg/entry.jl:461
 [6] (::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#update,Tuple{String,Set{String}}})() at ./pkg/dir.jl:36
 [7] cd(::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#update,Tuple{String,Set{String}}}, ::String) at ./file.jl:70
 [8] #cd#1(::Array{Any,1}, ::Function, ::Function, ::String, ::Vararg{Any,N} where N) at ./pkg/dir.jl:36
 [9] update() at ./pkg/pkg.jl:228

**Comment by ararslan** [comment]
Hm, weird. Can you try
```julia
cd(Pkg.dir("QuadGK")) do
    run(`git checkout master`)
    run(`git fetch origin --prune`)
    run(`git reset --hard origin/master`)
end
Pkg.free("QuadGK")
```
?

**Comment by goedman** [comment]
That fixed it! Thank you. I'll close this issue tomorrow.

**Comment by goedman** [comment]
By the way, this is on OS X:

julia> versioninfo()
Julia Version 0.6.1
Commit 0d7248e (2017-10-24 22:15 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i5-6267U CPU @ 2.90GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, skylake)

Also fixed the issue on Julia 0.7.--DEV.2803.

**Comment by ararslan** [comment]
If this is fixed I'll go ahead and close it now. Glad it's working again for you.
